carma_wm/src/CARMAWorldModel.cpp

```diff
@@ -613,15 +595,17 @@ namespace carma_wm
     return semantic_map_;
   }
 
+  #ifndef SONAR_SCAN
```

Comment:
nit: Can we add a comment here what these macros are for? Something simple like - Turn sonar scan on and stop sonar scan - should be good

Reply:
It didn't work, so removing